src/SingleStoreConnector/Core/ServerSession.cs

```diff
@@ -1011,7 +1011,11 @@ private async Task<bool> OpenTcpSocketAsync(ConnectionSettings cs, ILoadBalancer
 			try
 			{
 				ipAddresses = ioBehavior == IOBehavior.Asynchronous
+#if NET6_0_OR_GREATER
+					? await Dns.GetHostAddressesAsync(hostName, cancellationToken).ConfigureAwait(false)
+#else
 					? await Dns.GetHostAddressesAsync(hostName).ConfigureAwait(false)
+#endif
```

Comment:
This is already on Master for Original MySqlConnector https://github.com/mysql-net/MySqlConnector/blob/master/src/MySqlConnector/Core/ServerSession.cs#L1036